api_app/models/domain/authentication.py

```diff
@@ -13,3 +13,20 @@ class User(BaseModel):
     email: str = Field(None)
     roles: List[str] = Field([])
     roleAssignments: List[RoleAssignment] = Field([])
+
+
+class AssignableUser(BaseModel):
+    name: str
+    email: str
+
```

Comment:
Should we be using id rather than email t assign users throughout this code? We have had instances where users to do not have email addreses. Happy to discuss.

Reply:
This has now been implemented